spk/lidarr/src/service-setup-mono.sh

```diff
@@ -23,7 +23,7 @@ GROUP="sc-download"
 
 SERVICE_COMMAND="env PATH=${MONO_PATH}:${PATH} HOME=${HOME_DIR} LD_LIBRARY_PATH=${SYNOPKG_PKGDEST}/lib ${MONO} ${LIDARR}"
 SVC_BACKGROUND=y
-SVC_WAIT_TIMEOUT=60
+SVC_WAIT_TIMEOUT=90
```

Comment:
Is this file even needed anymore?

lidarr mono versions are EOL and unsupported.

lidarr v1 is dotnet and not mono

Reply:
> lidarr mono versions are EOL and unsupported.

Thanks for the hint. Another caveat is, that you need to install mono, as the repository tells the DSM Package Manager that it depends on mono, even when lidarr based on dotnet is installed.